src/components/Editor/ItemEditor.vue

```diff
@@ -204,17 +221,30 @@ export default {
       var millisecond = parseInt(timeInISO.millisecond) || 0;
       return hour * 3600 + minute * 60 + second + millisecond / 1000;
     },
-    updateCorrectOption(option) {
+    updateCorrectOption(optionNumber) {
```

Comment:
I had changed optionNumber to optionIndex everywhere - probably better to stay consistent with it (and I had made this change in my PR already :P)

Reply:
Will do when resolving conflicts - will be easier as it will be done in one click